gettsim/taxes/zu_verst_eink/freibetraege.py

```diff
@@ -258,15 +264,23 @@ def eink_st_sonderausgaben_ab_2012(
         12 * betreuungskost_m,
         eink_st_abzuege_params["kinderbetreuungskosten_abz_maximum"],
     )
+    sonderausgaben_betreuung = (
+        anz_kinder_tu
+        * abziehbare_betreuungskosten
+        * eink_st_abzuege_params["kinderbetreuungskosten_abz_anteil"]
+    ) / anz_erwachsene_tu
+
+    if gemeinsam_veranlagt_tu:
+        pauschale = eink_st_abzuege_params["sonderausgabenpauschbetrag"]["couple"]
```

Comment:
We should compare it only to ["sonderausgabenpauschbetrag"]["single"] as `sonderausgaben_betreuung` is also considered on the individual level. Or am I mistaken?

Reply:
I think it should be at the couple level, but that is mostly guessing from my own tax return.

To avoid the confusion I just had, it would be useful to define a variable

`sonderausgaben_gesamt = sonderausgaben_betreuung`

and do the comparisons below in terms of that one. Just to prepare for the future already.

Reply:
But `sonderausgaben` are currently defined on the individual level like all other freibeträge.

It might make sense to change it to the tax unit level, but I would not like to do it in this PR and rather postpone until we decided on #388.

Reply:
> To avoid the confusion I just had, it would be useful to define a variable...

I like this!

Reply:
> We should compare it only to ["sonderausgabenpauschbetrag"]["single"] as `sonderausgaben_betreuung` is also considered on the individual level. Or am I mistaken?

I think you are right. The `sonderausgaben_betreuung` was also scaled down to the individual level by dividing it by the number of 'anz_erwachsene_tu'. So I adjusted the comparison

> To avoid the confusion I just had, it would be useful to define a variable
> 
> `sonderausgaben_gesamt = sonderausgaben_betreuung`
> 
> and do the comparisons below in terms of that one. Just to prepare for the future already.

I changed the structure of the function a bit and added the suggested variable, so that we can add further items to the 'Sonderausgaben' in the future (e.g. donations, Kirchensteuer, etc.). That was the idea or did I misunderstand it?

Reply:
Looks good! Just one more comment